zebra-chain/src/chain_tip/tests/prop.rs

```diff
@@ -72,7 +72,7 @@ fn estimate_time_difference(
 ) -> Duration {
     let spacing_seconds = active_network_upgrade.target_spacing().num_seconds();
 
-    let height_difference = i64::from(end_height - start_height);
+    let height_difference = (end_height - start_height).map_or(0, |height| height.into());
```

Comment:
Nitpick/Optional:

```suggestion
    let height_difference = (end_height - start_height).map_or(0, i64::from);
```

Reply:
Turns out this is unnecessary after 4ad832e2e4